### PR TITLE
Cleanup (#2047): new license from OTM

### PR DESCRIPTION
**Pull request by ericjbohm**
*Original date: 2019-04-05 16:36:23*
*Original PR: https://charm.cs.illinois.edu/gerrit/5061*

---

specific description of non-commercial uses covered.

Change-Id: I6e316529bd71bc8681f8267910cf31fd156da692

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,148 +1,188 @@
...
+  of Computer Science, hereby grants You a non-commercial,
+  non-exclusive, non-transferable, royalty-free, restrictive license
+  to download, use, and create derivative works of the Software for
+  academic and research purposes only.
+
+License Restrictions.  Any uses of Software or Software derivative
+works beyond those granted in Section 2 above require a separate for a
+fee license.  To negotiate a license with additional rights, You may
+contact Illinois at the Office of Technology Management at
+otm@illinois.edu or Charmworks, Inc. at info@hpccharm.com."
```
Comment (matthiasdiener) [inline LICENSE]: Stray ".

File: LICENSE [hunk 2]
```diff
@@ -1,148 +1,188 @@
...
+
+15 Waiver.  The failure of either party to enforce any provision of
+   this License shall not constitute a waiver of that right or future
+   enforcement of that or any other provision.
+
+Approved for legal form Illinois Counsel SJA 10/18
+
+University Technology No. TF08023 Commercial Non-Exclusive Software License
+
+
```
Comment (matthiasdiener) [inline LICENSE]: Stray blank lines.

Comment (ericjbohm) [reply LICENSE]: Export from Word creates very strange issues. Should be addressed in the fixup patch.

**Comment by ericjbohm** [comment]
> Should we also add a (placeholder?) CITATION file to this PR?

Good point.  Do we have a template for one from the DOI registration site?

**Comment by mprobson** [comment]
I think @rbuch set that up but it may not generate until our next release. He can correct me if I'm wrong.

**Comment by ericjbohm** [comment]
see https://github.com/UIUC-PPL/charm/pull/2334 for example of CITATION new format.

**Comment by mprobson** [comment]
Great, no reason not to merge this then. Unless we want to merge citation at the same time.

**Comment by evan-charmworks** [comment]
Was this PR updated with the most recent version of the license as received from OTM after further discussion?

**Comment by ericjbohm** [comment]
> Was this PR updated with the most recent version of the license as received from OTM after further discussion?

It was updated with the change to the citation and a little clarification.  What outstanding changes are missing?

**Pull request merged**